src/rpc/README.md

```
@@ -105,34 +105,35 @@ new Subscription(subscriptionId).on('message', console.log)
 
-- [`method: commentUpdateSubscribe, params: [{cid: string}]`](#commentupdatesubscribe)
-- [`method: communityUpdateSubscribe, params: [{address: string}]`](#communityupdatesubscribe)
-- [`method: publishComment, params: [{comment, challengeAnswers, challengeCommentCids}]`](#publishcomment)
-- `method: publishVote, params: [{vote, challengeAnswers, challengeCommentCids}]`
-- `method: publishCommentEdit, params: [{commentEdit, challengeAnswers, challengeCommentCids}]`
-- `method: publishCommentModeration, params: [{commentModeration, challengeAnswers, challengeCommentCids}]`
-- `method: publishChallengeAnswers, params: [subscriptionId: number, {challengeAnswers}]`
-- `method: startCommunity, params: [{address: string}]`
+Each subscribe/publish method returns `{subscriptionId: number}` and streams JSON-RPC notifications until `unsubscribe` is called.
+
+- [`method: startCommunity, params: [{name?: string, publicKey?: string}]`](#startcommunity)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)